### PR TITLE
Contact pg

### PR DESCRIPTION
**Pull request by navyasingla1309**
**CONTACT US PAGE ADDED #12** 
1. contact page has been created.
2. Highlighted Active navlink.
3. Added contact url's as per reference.

here are the screenshots:
image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "next": "^11.0.0",
     "react": "^17.0.2",
     "react-animated-css": "^1.2.1",
+    "react-bootstrap": "^1.6.3",
```
Comment (ShreyaPrasad1209) [inline package.json:16]: Use [reactstrap](https://reactstrap.github.io/) for using bootstrap components and not react-bootstrap

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@navyasingla1309 close this PR and make the changes I've requested

File: package.json (hunk 1 above)
Comment (ShreyaPrasad1209) [reply package.json:16]: Remove the images. I'll send you a contact page for reference.

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
I have reverted your previous PR because the file structure was wrong. I missed it in my last review. Whatever components you need to add, make in inside components. Pages folder contains route to each page and therefore we should not create components like navbar, footer inside pages.

**Comment by navyasingla1309** [comment]
> I have reverted your previous PR because the file structure was wrong. I missed it in my last review. Whatever components you need to add, make in inside components. Pages folder contains route to each page and therefore we should not create components like navbar, footer inside pages.
Di, you reverted the PR of about page, I did the mentors page. But I get it, I will correct it in my current work

File: package.json (hunk 1 above)
Comment (navyasingla1309) [reply package.json:16]: > Remove the images. I'll send you a contact page for reference.

Should I remove all the images?  And what about the animations in the links?

Comment (navyasingla1309) [reply package.json:16]: > Use [reactstrap](https://reactstrap.github.io/) for using bootstrap components and not react-bootstrap

ok

**Pull request closed without merging**